docs/modules/operation/pages/quick-start/performance-data.adoc

```diff
@@ -0,0 +1,41 @@
+
+= Step 4: Set Up Performance Data Collection
+
+This section describes how to set up performance data collection in {page-component-title}.
+
+{page-component-title} collects performance data using xref:reference:daemons/daemon-config-files/collectd.adoc[collectd], which is enabled by default.
+Collectd schedules data collection on network entities using management agents and protocol-specific collectors.
+Data collection for SNMP and OpenNMS-JVM is enabled by default, allowing {page-component-title} to monitor itself through JMX.
+
+For additional information, including collector configuration, see xref:operation:deep-dive/performance-data-collection/introduction.adoc[Performance Management] in the Deep Dive section.
+
+== Before you begin
+
+{page-component-title} data collection for SNMP works out of the box, provided your SNMP string is configured correctly.
+
+The default value for the community string is `public`.
```

Comment:
Community string is used only for SNMPv1 and SNMPv2c. SNMPv3 uses a different mechanism. I know we want to avoid rabbit-holes in this guide, so maybe we add an info box to the effect:

If you are using SNMPv3, you will need to configure details for a USM (User-based Security Model) user instead of a community string. {page-component-title} fully supports SNMPv3 and USM, but these topics are outside the scope of this guide. Please see the [link to deep-dive] for more information.

Reply:
It looks like we don't have documentation about these topics as they relate to performance data gathering. I'll take a note and see if we should make a ticket to add them to the docs.

```suggestion
The default value for the community string is `public`.

NOTE: If you are using SNMPv3, you need to configure details for a User-Based Security Model (USM) user instead of a community string.
{page-component-title} supports SNMPv3 and USM, but these topics are outside of the scope of this guide.

```